packages/swarm-components/src/Checkbox.jsx

```diff
@@ -63,7 +63,7 @@ const Checkbox = (props: Props): React.Element<'label'> => {
 				aria-checked={checked}
 			>
 				{checked && (
-					<Icon shape="check" size="xs" color={disabled ? '#A2A2A2' : '#ffffff'} />
+					<Icon shape="check" size="xs" color={disabled ? 'var(--color-gray-6)': '#ffffff'} />
```

Comment:
I know color prop is supported by `Icon`, but wouldn't it be better to have this specified as a selector in the css file? Something like `[data-swarm-checkbox="disabled"] svg`?

Reply:
Yes, we will get there with these.